### PR TITLE
adds wallet server information to `status` command

### PR DESCRIPTION
**Pull request by jessopb**
Adds 
known_servers: int
available_servers: int 
to wallet status.

Also changes
connected_servers: {...}
to 
servers: {
[
host, port, latency, available
],
}

### CODE REVIEW

**Comment by jessopb** [comment]
An alternative to this solution is listing all possible "sessions" - those that are available, and those that aren't. 

Then the app ui would be able to tell the user which custom servers are misbehaving.

File: lbry/lbry/extras/daemon/Components.py [hunk 1]
```diff
@@ -107,25 +107,42 @@ def component(self):
...
+                        {
+                            'host': session.server[0],
+                            'port': session.server[1],
+                            'latency': round(session.connection_latency, 2) if session.connection_latency else None,
+                            'availability': session.available,
+                        } for session in sessions
+                    ],
+                    'known_servers': len(sessions),
+                    'available_servers': sum(s.available is True for s in sessions)
+                }
```
Comment (eukreign) [inline lbry/lbry/extras/daemon/Components.py]: Can you change the structure of the method to be:
```
result = # create the common response
if self.wallet_manager.ledger.network.remote_height:
   result[...] = # add the extra values available when we have remote_height
return result
```

File: lbry/lbry/extras/daemon/Components.py [hunk 2]
```diff
@@ -107,25 +107,42 @@ def component(self):
...
+                local_height = self.wallet_manager.ledger.local_height_including_downloaded_height
+                remote_height = self.wallet_manager.ledger.network.remote_height
+                best_hash = self.wallet_manager.get_best_blockhash()
+                progress = min(max(math.ceil(float(local_height) / float(remote_height) * 100), 0), 100)
+                return {
+                    'servers': [
+                        {
+                            'host': session.server[0],
+                            'port': session.server[1],
+                            'latency': round(session.connection_latency, 2) if session.connection_latency else None,
```
Comment (eukreign) [inline lbry/lbry/extras/daemon/Components.py]: Can this just be `'latency': session.connection_latency,`?

File: lbry/lbry/extras/daemon/Components.py [hunk 3]
```diff
@@ -107,25 +107,42 @@ def component(self):
...
+                            'availability': session.available,
+                        } for session in sessions
+                    ],
+                    'headers_synchronization_progress': progress,
+                    'blocks': max(local_height, 0),
+                    'blocks_behind': max(remote_height - local_height, 0),
+                    'best_blockhash': best_hash,
+                    'known_servers': len(sessions),
+                    'available_servers': sum(s.available is True for s in sessions)
```
Comment (eukreign) [inline lbry/lbry/extras/daemon/Components.py]: I don't know what this is doing... `sum()` is a function that adds numbers together... I would imagine that adding booleans together would be undefined behavior?

I think this should be: `len(s for s in sessions if s.available)`

**Pull request merged**